index.html

```diff
@@ -11,7 +11,7 @@
             <div><button onclick="reset()">Reset</button></div>
             <div><button onmousedown="startThrust()" onmouseup="stopThrust()">Thrust</button></div>
             <div id="fuel">fuel</div>
-            
+            <input type="range" id="a" value="50">
```

Comment:
This widget doesn't seem to do anything.  If you are looking for an interesting and simple way to make a gauge, take a look at the html meter widget

Reply:
ok ..